BetterHitboxStardew/BetterHitboxStardew.csproj

```diff
@@ -46,7 +52,6 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="CursorPosition.cs" />
```

Comment:
CursorPosition.cs removed as it was a duplicate of an existing method within SMAPI - duplicate methods was causing SMAPI to refuse to load the mod at runtime. Default SMAPI method is named the same and allows for the same functions.